Autopilot/Inc/SensorFusion.hpp

```diff
@@ -42,12 +65,22 @@ SFError_t SF_GenerateNewResult();
 
 /**
  * Get latest fused sensor data. Can be called any time data is needed after SF_init has been called once. Waits until the output struct is not being accessed by another task.
- * @return Output struct for fused data.
+ * @param [out] output Output struct for fused data.
+ * @return Error struct.
  */ 
-SFOutput_t SF_GetResult();
-
+SFError_t SF_GetResult(SFOutput_t *SFoutput);
 
+/**
+ * Get raw sensor data. Can be called any time raw data is needed.
+ * @return IMU struct.
+ */ 
+IMU_Data_t SF_GetRawIMU();
 
+/**
+ * Get raw sensor data. Can be called any time raw data is needed.
+ * @return Airspeed struct.
+ */ 
+Airspeed_Data_t SF_GetRawAirspeed();
```

Comment:
While we are at it, we should make similar methods for the altimeter and gps. That way I don't have to bug you in the future in case we need it